common/lib/xmodule/xmodule/vertical_block.py

```diff
@@ -81,6 +81,8 @@ def _student_or_public_view(self, context, view):
 
         # pylint: disable=no-member
         for child in child_blocks:
+            if context.get('hide_access_error_blocks') and getattr(child, 'has_access_error', False):
```

Comment:
I love Python, but I always get tripped up that `getattr` raises an exception by default if it doesn't exist.

Reply:
I'm a bit confused, here... We didn't add this code for the old server-side view, despite the fact that the old view and the MFE both render via the same vertical code. So, why do we need this now?

I think the answer is we don't, and that the code used for the old view which swapped in access denied messages using xblock wrappers should be sufficient, but that we need to inject the correct xblock wrappers in the call used by the MFE.

Reply:
So this is because we don't actually want to render the access denied messages since the Learning MFE renders a general Access denied message at the top level of the courseware (outside of the vertical iFrame). Without the inclusion of these lines and stopping the rendering of the xblocks that would render the access denied message, we would end up with repeated messaging inside of the Learning MFE.

Or are you saying something should exist at the point where the wrapper is being called to ignore it if coming from the MFE?

Reply:
Ah, ok, I understand. No, this is fine, if we're preserving the MFE placement of the message. Once we get to full-MFE, we can probably remove the existing xblock wrapper code.

Reply:
Yeah, I'm hoping a lot of this can be cleaned up once we are out of the iframe world. Totally agree with you.